src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -261,22 +261,7 @@ private synchronized void init() {
         if (views.size() == 0) {
             ListView lv = new ListView("All", this);
             views.add(lv);
-            try {
-                try {// HACK ALERT: ListView doesn't expose us a method to set this as of 1.480 yet,
-                    // so we forcibly do it.
-                    Field f = ListView.class.getDeclaredField("includeRegex");
-                    f.setAccessible(true);
-                    f.set(lv, ".*");
-                    f = ListView.class.getDeclaredField("includePattern");
-                    f.setAccessible(true);
-                    f.set(lv, Pattern.compile(".*"));
-                } catch (Throwable e) {
-                    LOGGER.log(Level.WARNING, "Initial view may be configured incorrectly", e);
-                }
-                lv.save();
-            } catch (IOException e) {
-                LOGGER.log(Level.WARNING, "Failed to set up the initial view", e);
-            }
```

Comment:
This part of the change is not relevant to this PR, right? Looks as though it's just repairing an earlier issue.